Doc/library/sys.rst

```diff
@@ -338,12 +338,13 @@ always available.
 
    This function prints out a given traceback and exception to ``sys.stderr``.
 
-   When an exception is raised and uncaught, the interpreter calls
-   ``sys.excepthook`` with three arguments, the exception class, exception
-   instance, and a traceback object.  In an interactive session this happens just
-   before control is returned to the prompt; in a Python program this happens just
-   before the program exits.  The handling of such top-level exceptions can be
-   customized by assigning another three-argument function to ``sys.excepthook``.
+   When an exception other than :exc:`SystemExit` is raised and uncaught,
+   the interpreter calls ``sys.excepthook`` with three arguments, the
```

Comment:
Since the lines after the addition of `SystemExit` don't change, I'd say just make a new line even though it might be shorter than the rest

Reply:
Good idea, I've pushed to un-reflow this. (Also updated the branch from `main`.)